plugins/modules/win_acl.ps1

```diff
@@ -104,6 +107,9 @@ if ($path_qualifier -eq "HKU:" -and (-not (Test-Path -LiteralPath HKU:\))) {
 if ($path_qualifier -eq "HKCC:" -and (-not (Test-Path -LiteralPath HKCC:\))) {
     New-PSDrive -Name HKCC -PSProvider Registry -Root HKEY_CURRENT_CONFIG > $null
 }
+if ($path_qualifier -eq "Cert:" -and (-not (Test-Path -LiteralPath Cert:\))) {
```

Comment:
you're right, it doesn't! in fact, I had to remove this bit from the test (in the fact where we test for effective permissions) because double mounting was making the script crash. that's interesting that it silently fails here. good catch!